docs/man/dune.1.md

```diff
@@ -0,0 +1,285 @@
+---
+title: dune
+section: 1
+header: User Manual
+footer: dune 1.1.0
```

Comment:
```suggestion
footer: AntelopeIO
```
The following three options all seem acceptable to me:
- Organization: "AntelopeIO"
- Version: "1.1.0" or "v1.1.0"
- name + ver: "DUNE 1.1.0"

@iamveritas What do you think? Personally, I like organization best. Best practices are unclear which is to be selected; however, a really quick review of existing man pages on my machine seems to indicate all of the above are acceptable.

Reply:
Upon further consideration, we should avoid version information in the manpage unless we auto generate it. There is too much risk we will forget to update it and there is no requirement to include version.

Reply:
With that, I recommend we use the organization: `footer: AntelopeIO`.